packages/@aws-cdk/aws-lambda/lib/scalable-version-attribute.ts

```diff
@@ -0,0 +1,65 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+
+/**
+ * Interface for scalable attributes
+ */
+export interface IScalableVersionAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level
+   */
+  scaleOnUtilization(props: UtilizationScalingProps): void;
+  /**
+   * Add scheduled scaling for this scaling attribute
+   */
+  scaleOnSchedule(id: string, actions: appscaling.ScalingSchedule): void;
+}
+
+/**
+ * Properties for enabling Lambda utilization tracking
+ */
+export interface UtilizationScalingProps extends appscaling.BaseTargetTrackingProps {
+  /**
+   * Target utilization percentage for the attribute
+   */
+  readonly targetUtilizationPercent: number;
+}
+
+/**
+ * A scalable lambda version attribute
+ */
+export class ScalableVersionAttribute extends appscaling.BaseScalableAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level
+   */
+  public scaleOnUtilization(props: UtilizationScalingProps) {
+    if (props.targetUtilizationPercent < 0 || props.targetUtilizationPercent > 1) {
+      throw new Error('target utilization is a percentage and must be between 0 and 1.');
```

Comment:
Add more information to this error, the calling user does not know which metric is tracked (`LAMBDA_PROVISIONED_CONCURRENCY_UTILIZATION `) so it is unclear why the value should be in this value range

Reply:
Done